js/MetricQWebView.js

```diff
@@ -82,7 +83,7 @@ class MetricQWebView {
       }
     }
 
-    this.updateMetricUrl()
+    //this.updateMetricUrl()
```

Comment:
Despite the bad examples around, just remove code that is not used. If you want to leave a comment, it should tell something specific.